catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'frontend-app-shell'
+  description: "A Piral shell application for use with Open edX MFEs."
+  links:
+    - url: "https://github.com/ichintanjoshi/frontend-app-shell"
+      title: "frontend-app-shell"
+      icon: "GitHub"
```

Comment:
This is the wrong link and we don't need to link to the github repo that happens automatically.

Reply:
@feanil  have updated